MPM/mpm_gateway.php

```diff
@@ -174,6 +174,12 @@ public function process_payment($order_id)
 
 		$order->update_status('pending', __('Awaiting payment confirmation', 'MPM'));
 
+		$isCancelled = get_post_meta($order->id, '_is_mollie_cancelled');
+		if ($isCancelled)
+		{
+			update_post_meta($order_id, '_is_mollie_cancelled', 0, 1);
```

Comment:
Hier graag nog het volgende toevoegen zodat het in de admin duidelijk is dat de betaling is voorgezet na annulering.

`$order->add_order_note("User retried payment");`